File: app/src/main/java/dk/kea/androidgame/martin/myfirstgameengine/TestGame.java
package dk.kea.androidgame.martin.myfirstgameengine;

import dk.kea.androidgame.martin.myfirstgameengine.core.GameEngine;
import dk.kea.androidgame.martin.myfirstgameengine.core.Screen;

public class TestGame extends GameEngine
{
    @Override
    public Screen createStartScreen()
    {
        return new TestScreen(this);
    }
}

File: app/src/main/java/dk/kea/androidgame/martin/myfirstgameengine/TestGame.kt
package dk.kea.androidgame.martin.myfirstgameengine

import dk.kea.androidgame.martin.myfirstgameengine.core.GameEngine
import dk.kea.androidgame.martin.myfirstgameengine.core.Screen

class TestGame : GameEngine() {
    override fun createStartScreen(): Screen {
        return TestScreen(this)
    }
}
